### DISCUSSION TITLE
Quarterly periods

### DISCUSSION THREAD

**Discussion by Kims2009** [flixlix/energy-period-selector-plus#14] [Ideas]
Will it be possible to add a "fixed" period button so the showed consumption will be per 3 month eg. 1.1.-31.3.? Found this hacs on Bearded Thinker 😀 Great work

**Comment by flixlix** [comment]
Hi, I'm glad you like my card 😊
This would add quite a lot of complexity, because this would have to be selectable through the card configuration. I am going to think about how to implement this. In the meantime, feel free to create a PR where you implement this yourself

**Comment by Kims2009** [comment]
Hi again, I do not know anything about programming (unfortunately) so I
guess that leaves it up to you how to figure it out 🧐

Mayby  it should be implemented directly in HA?

Kim

tir. 1. aug. 2023 13.51 skrev Luca Félix ***@***.***>:
